.actions/assistant.py

```diff
@@ -240,6 +244,7 @@ def _load_aggregate_requirements(req_dir: str = "requirements", freeze_requireme
     requires = [
         load_requirements(d, unfreeze="none" if freeze_requirements else "major")
         for d in glob.glob(os.path.join(req_dir, "*"))
+        if os.path.exists(os.path.join(d, "base.txt"))
```

Comment:
Does this project not have base requirements? Maybe it would be better to create an empty file for consistency